### PR TITLE
DFPL-2236: add child details to smoke test

### PR DESCRIPTION
**Pull request by abigailsmith97**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-2236

### Change description ###
adds child details to E2E smoke test 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: playwright-e2e/tests/smoke-test.spec.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { test, expect } from "../fixtures/fixtures";
 import { BasePage } from "../pages/base-page";
+//import { ChildDetails } from "../pages/child-details";
```
Comment (Iphiclus) [inline playwright-e2e/tests/smoke-test.spec.ts:3]: Hi, can we delete this?

**Review by Iphiclus — CHANGES_REQUESTED** [review]

File: playwright-e2e/tests/smoke-test.spec.ts (hunk 1 above)
Comment (abigailsmith97) [reply playwright-e2e/tests/smoke-test.spec.ts:3]: yes, will remove now!

File: playwright-e2e/fixtures/create-fixture.ts [hunk 2]
```diff
@@ -115,8 +118,15 @@ export const test = base.extend<CreateFixtures>({
     await use(new CaseDetails(page));
   },
 
+  childDetails: async ({ page }, use) => {
+    await use(new ChildDetails(page));
+  },
+
+  basePage: async ({ page }, use) => {
```
Comment (GauravChawlaMOJ) [inline playwright-e2e/fixtures/create-fixture.ts:125]: is there a reason why we need basePage here?

Comment (abigailsmith97) [reply playwright-e2e/fixtures/create-fixture.ts:125]: just pulling the branch now and removing from fixture to see - can't quite remember why i added it..

Comment (abigailsmith97) [reply playwright-e2e/fixtures/create-fixture.ts:125]: yep, not needed -- deleted now

**Review by Iphiclus — APPROVED** [review]

**Pull request merged**